charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```
@@ -171,6 +171,14 @@ spec:
 {{- end }}
 {{- if .Values.global.operator.watchdogCanPatchPodLabels }}
         - "--watchdog-can-patch-pod-labels"
+{{- end }}
+{{- if .Values.global.actors.enabled }}
```

Should this be `.serviceName`?

```suggestion
{{- if .Values.global.actors.serviceName }}
```

---

Tried to check `enabled` actually for actors and thought that if `enabled` is true, then serviceName is provided. But, I am fine checking serviceName directly. Done the change.